tests/by-util/test_ls.rs

```diff
@@ -5716,3 +5716,37 @@ fn test_unknown_format_specifier() {
         .succeeds()
         .stdout_matches(&re_custom_format);
 }
+
+#[cfg(all(unix, not(target_os = "macos")))]
+#[test]
+fn test_acl_display_symlink() {
+    use std::process::Command;
+
+    let (at, mut ucmd) = at_and_ucmd!();
+    let dir_name = "dir";
+    let link_name = "link";
+    at.mkdir(dir_name);
+
+    // calling the command directly. xattr requires some dev packages to be installed
+    // and it adds a complex dependency just for a test
+    match Command::new("setfacl")
+        .args(["-d", "-m", "u:bin:rwx", dir_name])
```

Comment:
I think you have to use `at.plus_as_string`:
```suggestion
        .args(["-d", "-m", "u:bin:rwx", &at.plus_as_string(dir_name)])
```
At least on my system the `setfacl` command fails otherwise and the `Ok(_)` arm is executed.